packages/actions-global/src/hooks.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {
+  showNotification,
+  showUnexpectedErrorNotification,
+  showApiErrorNotification,
+} from './global';
+import { useDispatch } from 'react-redux';
+
+// Returns a function that dispatches an API error notification.
+// Example:
+//   const showApiErrorNotification = useShowApiErrorNotification();
+//   showApiErrorNotification({ errors });
+export function useShowApiErrorNotification() {
+  const dispatch = useDispatch();
+  return React.useCallback(
+    (...args) => dispatch(showApiErrorNotification(...args)),
+    [dispatch]
+  );
+}
+
+// Returns a function that dispatches an unexpected error notification.
+// Example:
+//   const showUnexpectedErrorNotification = useShowUnexpectedErrorNotification();
+//   showUnexpectedErrorNotification({ error });
+export function useShowUnexpectedErrorNotification() {
+  const dispatch = useDispatch();
+  return React.useCallback(
+    (...args) => dispatch(showUnexpectedErrorNotification(...args)),
+    [dispatch]
+  );
+}
+
+// Returns a function that dispatches a notification, pre-configured to
+// a speficic notification.
+// Example:
+//   const showSuccessNotification = useShowNotification({
+//     kind: 'success',
+//     domain: DOMAINS.SIDE,
+//   });
+//   showSuccessNotification({ text: "All good!" });
+export function useShowNotification(notificationFragment) {
```

Comment:
I struggled with naming this too. Fragment sounds good to me. I think I went with `config` and `content` in the end.

Reply:
Yeah, I find `config` a bit misleading though, as it seems that you can pass some configuration options (like when you're setting up a server or a request or something).
In reality is just a part of the notification object, so I figured "notification fragment" to be a more suitable name.